WooCommerce/Classes/ViewRelated/Dashboard/Stats v3/DeprecatedStatsDashboardViewController.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import UIKit
+
+/// Empty state screen shown when the store stats version is not supported
+///
+final class DeprecatedDashboardStatsViewController: UIViewController {
+
+    /// Empty state screen
+    ///
+    private lazy var emptyStateViewController = EmptyStateViewController(style: .basic)
+
+    /// Empty state screen configuration
+    ///
+    private let emptyStateConfig: EmptyStateViewController.Config = {
+        let message = NSAttributedString(string: Constants.title, attributes: [.font: EmptyStateViewController.Config.messageFont.bold])
+        return EmptyStateViewController.Config.withLink(message: message,
+                                                        image: .noStoreImage,
+                                                        details: Constants.details,
+                                                        action: .text(title: Constants.buttonTitle, linkURL: WooConstants.helpCenterURL))
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        displayEmptyViewController()
+    }
+
+    /// Shows the EmptyStateViewController
+    ///
+    func displayEmptyViewController() {
+        addChild(emptyStateViewController)
+
+        emptyStateViewController.view.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(emptyStateViewController.view)
+
+        emptyStateViewController.view.pinSubviewToAllEdges(self.view)
+        emptyStateViewController.didMove(toParent: self)
+        emptyStateViewController.configure(emptyStateConfig)
+    }
+}
+
+// MARK: DashboardUI conformance
+// Everything is empty as this deprecated stats screen is static
+extension DeprecatedDashboardStatsViewController: DashboardUI {
+
+    var displaySyncingErrorNotice: () -> Void {
+        get {
+            return {}
+        }
+        set {}
+    }
+
+    var onPullToRefresh: () -> Void {
+        get {
+            return {}
+        }
+        set {}
+    }
+
+    func defaultAccountDidUpdate() {}
+    func remindStatsUpgradeLater() {}
+    func reloadData(completion: @escaping () -> Void) {}
+}
+
+// MARK: Constants
+private extension DeprecatedDashboardStatsViewController {
+    struct Constants {
+        static let title = NSLocalizedString("We can’t display your store’s analytics",
+                                      comment: "Title when we can't show stats because user is on a deprecated WC Version")
+        static let details = NSLocalizedString("""
+                                        Make sure you are running the latest version of WooCommerce on your site and that you have
+                                        WooCommerce Admin activated.
```

Comment:
Hmmm, I guess that since you can [disable the plugin by using an extension](https://wordpress.org/plugins/disable-dashboard-for-woocommerce/) it would make sense the "and" word, but my English may be failing me! 😛 

I'll let @Garance91540 confirm!

Reply:
ah interesting that [this third-party plugin](https://wordpress.org/plugins/disable-dashboard-for-woocommerce/) disables WC Admin that is included in WC 4.0+. before WC 4.0, the dashboard is available by installing and activating the [official WC Admin plugin](https://wordpress.org/plugins/woocommerce-admin/). I could see that either "and" or "or" work depending on why the site doesn't have WC Admin activated. up to you and @Garance91540!